pokeFormStyle.css

```diff
@@ -0,0 +1,34 @@
+body {
```

Comment:
don't forget * {
box-sizing: border-box;
}

and on the body font-sizing: 62.5%